graphene_django/compat.py

```diff
@@ -1,10 +1,12 @@
 import sys
+from collections.abc import Callable
 from pathlib import PurePath
 
 # For backwards compatibility, we import JSONField to have it available for import via
 # this compat module (https://github.com/graphql-python/graphene-django/issues/1428).
 # Django's JSONField is available in Django 3.2+ (the minimum version we support)
```

Comment:
minor: this comment should appear above the import of JSONField instead of above `import django`

Reply:
huh, it was an unused import of django, but the IDE showed it similar to the comment above, so I didn't notice it
<img width="703" alt="image" src="https://github.com/user-attachments/assets/2c389854-7f07-4a81-8168-4367e7f42d07" />

remove the import.

Though, do you think it would make sense to add a linter rule for unused imports? (in a separate PR)

Reply:
Looks like the unused imports lint rule was disabled for the whole file https://github.com/graphql-python/graphene-django/blob/8d4a64a40dca6b2c0b355bdb89b9205309884deb/.ruff.toml#L28 but we should probably remove that and instead disable it on a per-import basis

Reply:
removing the ruff rule for the compat.py file didn't break linting